operators/src/processing/reprojection.rs

```diff
@@ -555,7 +555,7 @@ where
                 ctx,
                 sub_query_spec,
             )
-            .filter_and_fill())
+            .filter_and_fill(CacheExpiration::NoCache)) // TODO: if we were sure that gaps only occur where the projection is not defined in the input, we could cache this forever
```

Comment:
hm, we should think about something here

Reply:
9b5283436f1ac6f6344a5396a0d0ddda4be63040